src/um/wia.rs

```diff
@@ -0,0 +1,756 @@
+// Copyright © 2015-2017 winapi-rs developers
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms
+use shared::basetsd::ULONG64;
+use shared::guiddef::{GUID, LPGUID, REFCLSID, REFGUID, REFIID};
+use shared::minwindef::{BOOL, BYTE, DWORD, FILETIME, LPBYTE, LPDWORD, LPLONG, PULONG, ULONG};
+use shared::windef::{HWND, RECT};
+use shared::wtypes::{BSTR, LPBSTR, PROPID};
+use shared::wtypesbase::LPOLESTR;
+use um::objidl::STGMEDIUM;
+use um::objidlbase::IStream;
+use um::propidl::{IEnumSTATPROPSTG, PROPSPEC, PROPVARIANT, STATPROPSETSTG};
+use um::unknwnbase::{IUnknown, IUnknownVtbl};
+use um::winnt::{HRESULT, LONG, VOID};
+STRUCT!{struct WIA_DITHER_PATTERN_DATA {
+    lSize: LONG,
+    bstrPatternName: BSTR,
+    lPatternWidth: LONG,
+    lPatternLength: LONG,
+    cbPattern: LONG,
+    pbPattern: *const BYTE,
```

Comment:
```suggestion
    pbPattern: *mut BYTE,
```
C header does not specify const here.